occjava/src/BRepOffsetAPI.i

```diff
@@ -54,3 +56,20 @@ class BRepOffsetAPI_NormalProjection: public BRepBuilderAPI_MakeShape
 	const TopoDS_Shape& Ancestor(const TopoDS_Edge& E) const;
 	Standard_Boolean BuildWire(TopTools_ListOfShape& Liste) const;
 };
+
+
+class BRepOffsetAPI_MakeOffsetShape : public BRepBuilderAPI_MakeShape {
+};
+
+class BRepOffsetAPI_MakeThickSolid  : public BRepOffsetAPI_MakeOffsetShape {
+public:
+	BRepOffsetAPI_MakeThickSolid(
+		const TopoDS_Shape& S, 
+		const TopTools_ListOfShape& ClosingFaces, 
+		const Standard_Real Offset, 
+		const Standard_Real Tol, 
+		const BRepOffset_Mode Mode = BRepOffset_Skin, 
```

Comment:
BRepOffset_Mode is not wrapped so I guess you don't need it. So all optional argument should be removed here. Wrapping BRepOffset_Mode and adding other argument could be done later.